packages/esm-patient-banner-app/src/banner-tags/deceased-tag.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import DeceasedBannerTag from './deceased-tag.component';
+import { mockDeceasedPatient } from '../../../../__mocks__/mockDeceasedPatient';
+import { mockPatient } from '../../../../__mocks__/patient.mock';
+
+describe('deceasedTag', () => {
+  it('renders a deceased tag in the patient banner for patients who died', () => {
+    render(<DeceasedBannerTag patient={mockDeceasedPatient} />);
+
+    expect(screen.getByRole('button', { name: /Deceased/ })).toBeInTheDocument();
+  });
+
+  it('doesnot render Deceased tag for patients who are still alive', () => {
+    render(<DeceasedBannerTag patient={mockPatient} />);
+
+    expect(screen.queryByRole('button', { name: /Deceased/ })).not.toBeInTheDocument();
```

Comment:
Is there any reason why you used `queryByRole` here and `getByRole` on the test before?

Reply:
When I used QueryByRole I was trying to assert that an element (the Deceased tag) will not be present when the patient.deceasedBoolean is false. I did not use getByRole in the second instance because the element does not show up and It would throw an error if no elements match.

@dkayiwa any suggestions

Reply:
@nanfuka You are perfectly correct. This is a good way to use these queries.